charts/plugin-health-scoring/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v2
 description: A Helm chart for Kubernetes
 name: plugin-health-scoring
 type: application
-version: 2.3.14
+version: 2.3.15
```

```suggestion
version: 3.0.0
```

Default UID and GID changed: it is a breaking change